rspace/src/test/scala/coop/rchain/rspace/HistoryActionsTests.scala

```diff
@@ -36,6 +36,14 @@ trait HistoryActionsTests
 
   type TestGNAT = GNAT[String, Pattern, String, StringsCaptor]
 
+  def getRootHash(store: IStore[String, Pattern, String, StringsCaptor],
```

Comment:
It might be good to add a comment that this a convenience function for testing.

Reply:
and `private[rspace]` access level?

Reply:
Done.